_posts/2019-05-15-vvv-3-0-0.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: default
+title:  "VVV 3.0"
+date:   2019-05-15 18:00:00 -0000
+author: tomjn
+---
+
+
+Hi! Welcome to the very late release post for VVV <a href="https://github.com/Varying-Vagrant-Vagrants/VVV/releases/tag/3.0.0"> v3.0</a>. As we follow semantic versioning, this is a breaking release, and requires slightly different update instructions.
+
+## Ubuntu 14 EOL and moving to Ubuntu 18 LTS
+
+VVV 2 uses Ubuntu 14, but this reached end of life recently. In response Ondrej deleted the unsupported PHP packages, and all VVV 2 installs were no longer able to provision. VVV contributors had setup a package mirror in case this had happened, but it seems only 70% of the needed packages were mirrored.
+
+For this reason, VVV 2 is no longer supported, and won't be recieving fixes and updates.
+
+VVV 3 moves us to a new Ubuntu 18.04 LTS box, which should cover us until 2024. Additionally, we're using a leaner custom built box. This does mean that you will need to destroy your VM to update:
+
+```
+# turn off the VM
+vagrant halt
+# destroy the Ubuntu 14 VM
+vagrant destroy
+# pull down the latest update
+git pull
+# provision a shiny new Ubuntu 18 VM
+vagrant up --provision
+```
+
+This will destroy the database, but luckily VVV has been creating backups for a long time, you can restore them after updating with this command:
+
+```
+vagrant ssh -c "db_restore"
+```
+
+If you turned off database backups, you can manually back up your database before updating with this command:
+
+```
+vagrant ssh -c "db_backup"
+```
+
+## What Else Has Changed?
+
+### MariaDB data No Longer Inside the VM
+
+VVV3 stores its database in `database/data` now, which means it will survive a `vagrant destroy`. If you're having trouble provisioning mariadb, let us know on github issues.
+
+We also added a config option to return to the old behaviour for those having problems:
+
+```
+general:
+  db_share_type: false
+```
+
+We continued optimising provision in this release, and making VVV friendlier.
+
+### Vagrant Guest Additions Plugin Disabled
+
+Our new box has recent guest additions, so this shouldn't be needed. However, some users like this plugin, and when reprovisioning encountered problems. We've disabled the plugin just for VVV to avoid these issues.
```

Comment:
I think that add the command to remove it can be useful

Reply:
it should be ok, we tell the plugin not to run if it's present anyway, they might be able to fix it at their end too in the future